test/smoke.spec.ts

```diff
@@ -7,7 +7,7 @@ describe('smoke tests', () => {
   it('can do `snyk version`', async () => {
     const imageName = process.env.IMAGE_TAG;
     const { stdout, stderr } = await runContainer(imageName, 'snyk version');
-    const versionRegex = /1\.\d\d\d\d\.0 \(standalone\)/;
+    const versionRegex = /1\.\d{4}\.0/;
```

Comment:
Thought/Suggestion: the regular expression is very limiting, since it requires 1.xxxx.0, how about fixing the expression even more by using something like this https://github.com/snyk/cli/blob/master/cliv2/main_integration_test.go#L27 ?

Reply:
Done